advanced_samples/Bulk_Upload/DotNet/Bulk_Uploader.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30104.148
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Bulk_Uploader", ".\BulkUploader\Bulk_Uploader.csproj", "{E0D4786B-CE18-4715-878A-BEB91FDD4455}"
```

Comment:
I've been adding a "Solution Items" folder to the solution files that contains the README.md, HISTORY.md, azure-pipelines.yml, polaris.yml, stylecop/fxcop files, etc.